scripts/populate_panels/load_g2p_data.pl

```diff
@@ -990,3 +998,88 @@ sub add_public_comments {
   return $count;
 
 }
+
+sub add_ontology_accession {
+   my ($disease, $disease_mim, $disease_mondo) = @_;
+   my @mondo = get_ontology_accession($disease_mim, $disease_mondo);
+   if (scalar @mondo > 0){
+     my $attribute = "Data source";
```

Comment:
If we import the MONDO accession supplied, we can say it comes from the submitter/data source. If we obtain it using OLS we need to say we got it from OLS. get_ontology_accession may do either here. 
We can start by only importing supplied MONDO accessions in the first instance.